### PR TITLE
Code quality exercise 3

### PR DESCRIPTION
**Pull request by gsngdavid**
Improve  functions

### CODE REVIEW

File: src/exercise3/exercise3.js
Comment (kareraolivier) [inline src/exercise3/exercise3.js:1]: the split operator on small line of code can be good

File: src/exercise3/exercise6.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function maxNumber(a, b, c) {
+  if (a > b && a > c) return a;
+  if (b > a && b > c) return b;
+  return c;
```
Comment (kareraolivier) [inline src/exercise3/exercise6.js:4]: Math.max makes the job more simple

**Review by kareraolivier — APPROVED** [review]
## This great code are improved and well optimized

File: src/exercise3/exercise6.js (hunk 1 above)
Comment (gsngdavid) [reply src/exercise3/exercise6.js:4]: I thought we're not allowed to change the logic

**Pull request merged**